### PR TITLE
docs/dasharo-menu-docs/overview.md: update menu screenshot

### PR DESCRIPTION
**Pull request by wiktormowinski**

### CODE REVIEW

**Comment by macpijan** [comment]
`@wiktormowinski
[Merge branch 'master' into update-menu-password](https://github.com/Dasharo/docs/pull/942/commits/a16389a56768a6ae2144430ec44e32e08a608f64)` please prefer rebase, not merge, when updating your branch

**Comment by wiktormowinski** [comment]
@macpijan sure, should be all clean now

File: docs/dasharo-menu-docs/overview.md [hunk 1]
```diff
@@ -129,7 +129,11 @@ possible feature extension of your platform.
 
 User Password Management menu allows one to set firmware setup admin password:
 
+<<<<<<< HEAD
```
Comment (filipleple) [inline docs/dasharo-menu-docs/overview.md:132]: @wiktormowinski yeah this shouldn't be here please fix

Comment (wiktormowinski) [reply docs/dasharo-menu-docs/overview.md]: idk how did it even get past through both `rebase --continue` and `pre-commit` X.X

**Review by filipleple — APPROVED** [review]

**Pull request merged**